src/obs-plugin/plugin-main.mm

```diff
@@ -108,8 +120,50 @@ bool obs_module_load(void)
             action->setText(obs_module_text("Start Virtual Camera"));
             obs_output_stop(output);
         } else {
+            
+            obs_video_info ovi;
+            obs_get_video_info(&ovi);
```

Comment:
Coding style nitpick: use spaces around operators (`!=` in this instance) like you do on line 139 for consistency with the rest of this codebase